src/main/bash/sdkman-init.sh

```diff
@@ -34,6 +34,8 @@ SDKMAN_PLATFORM="$(uname)"
 if [[ "$SDKMAN_PLATFORM" == 'Linux' ]]; then
 	if [[ "$(uname -m)" == 'i686' ]]; then
 		SDKMAN_PLATFORM+='32'
+	elif [[ "$(uname -m)" == 'aarch64' ]]; then
```

Comment:
Can we expand this to include 32-bit ARM too?

Reply:
I believe it is `armv7l`. However, it is a bit more complicated with ARM32 as there is another dimension of architecture that support hardware floating points (hf) vs software floating points (sf). I don't currently have a system to test this, so didn't want go in this. ARM64 (AArch64) is a little bit easier as hf included in that architecture.

Reply:
Just thinking out loud, we can also concatenate the OS and architecture here and send that to the server. This way we don't need to update the client and supporting a new architecture can be by adding support for it on the server side.

Reply:
Not completely sure if I understand, would you be able to give an example of what you mean?

Reply:
Instead of converting the platform here we can juts return `"$(uname)_$(uname -m)"` here and provide that to the server. For example,
- `Linux_x86_64` instead of `Linux64`
- `Linux_i686` instead of `Linux32`
- `Linux_aarch64` instead of `LinuxARM64`

But is see the will tightly couple it to `uname` which might be less desirable.

Reply:
Okay, got it. The problem with this is that we need to remain backward compatible. This would result in some nasty business on the server side.

That said, we could possibly do it but I would prefer if we handle this completely on the client side.

Reply:
We can leave this for a later discussion, I think this one is good to go for now.